src/resources/home/view.html

```diff
@@ -28,15 +28,15 @@ <h3>People</h3>
     <section class="row qg-cards">
         <article class="qg-card col-12 col-sm-6 col-lg-4">
             <div class="content">
-                <img src="https://via.placeholder.com/190x186" alt="">
+                <img src="https://www.qld.gov.au/dev/food-pantry/dev/kyfb-home-images/Skills-and-knowledge-176606698.jpg" alt="">
```

Comment:
should not be /dev/ or is changed when promoted to prd.

Reply:
Environments are being handled in the next sprint.